.github/workflows/chaos-tests.yml

```
@@ -0,0 +1,65 @@
+name: Chaos tests
```

Since we are accumulating so many workflows, would it make sense to name this so that it sorts near `Integration Tests`? e.g. `integration-chaos.yml` and something like:
```suggestion
name: Integration Chaos Test
```
:shrug: